setup.py

```diff
@@ -30,31 +30,19 @@ def run_tests(self):
 
 LONG_DESCRIPTION = README + '\n\n' + HISTORY
 
-INSTALL_REQUIRES_REPLACEMENTS = {
-    'https://github.com/ethereum/ethash/tarball/master#egg=pyethash': 'pyethash',
+# requirements
+install_requires = set(x.strip() for x in open('requirements.txt'))
+install_requires_replacements = {
+    'https://github.com/ethereum/serpent/tarball/develop': 'ethereum-serpent',
+    'https://github.com/ethereum/pydevp2p/tarball/develop': 'devp2p',
 }
+install_requires = [install_requires_replacements.get(r, r) for r in install_requires]
 
-INSTALL_REQUIRES = list()
-with open('requirements.txt') as requirements_file:
-    for requirement in requirements_file:
-        dependency = INSTALL_REQUIRES_REPLACEMENTS.get(
-            requirement.strip(),
-            requirement.strip(),
-        )
-
-        INSTALL_REQUIRES.append(dependency)
-
-INSTALL_REQUIRES = list(set(INSTALL_REQUIRES))
-
-DEPENDENCY_LINKS = []
-if os.environ.get("USE_PYETHEREUM_DEVELOP"):
```

Comment:
If `USE_PYETHEREUM_DEVELOP` flag has been removed, we also need to remove `USE_PYETHEREUM_DEVELOP=1` in `.travis.yml`.

Reply:
Will remove the line in .travis.yml.